.gitlab-ci.yml

```diff
@@ -44,6 +44,7 @@ lint:
     - if: $CI_EXTERNAL_PULL_REQUEST_IID
     - if: $NIGHTLY_TESTS
   script:
+    - pip install -r ci/requirements.txt
```

Comment:
This seems unfortunate, since we already specify the mypy dependency in 2 places (dev extras in setup.py and build system depends in pyproject.toml), but neither of these can be used here because I think we want to avoid building the package (this would add significant time to the linting stage)

Reply:
Looks like this is no longer required, since it is in the docker container?

Reply:
Good catch; removed in 829e253